src/data/faq.json

```diff
@@ -188,12 +188,11 @@
                             "<ul><li>the respective laboratory is not fully linked with the Corona Warn App backend.</li><li>the consent field allowing the transmission of the test result to the Corona-Warn-App was not ticked on the submission form (10C/OEGD) sent by your doctor to the laboratory.</li></ul>",

```

Comment:
"Corona Warn App" => "Corona-Warn-App" like in the following line for naming consistency.

Reply:
fixed